pages/guide/08-start.md

```diff
@@ -0,0 +1,422 @@
+      '''
+      dbConnectionName: Presto Data Source
+      friendlyName: Player Stats
+      description:
+      '''
+      A long description
+      '''
+      category: Sports
+      cardinality : large
+      readAccess : '(user AND member) OR (admin.user AND NOT guest user)'
+      isFact : true
+      tags: ['Game', 'Player']
+      measures : [
+          {
+          name : highScore
+          type : INTEGER
+          definition: 'MAX({{highScore}})'
+          friendlyName: High Score
+          }
+      ]
+      dimensions : [
+          {
+              name : name
+              type : TEXT
+              definition : '{{name}}'
+              cardinality : large
+          },
+          {
+              name : gameType
+              type : TEXT
+              definition : '{{game_type}}'
+              friendlyName: Game Type
+          },
+          {
+              name : createdOn
+              type : TIME
+              definition : '{{created_on}}'
+              grain:
+              {
+                  type :  DAY
+                  sql :  '''PARSEDATETIME(FORMATDATETIME({{}}, 'yyyy-MM-dd'), 'yyyy-MM-dd')'''
+              }
+          }
+      ]
+  }]
+}
+```
+
+For more information on subqueries please check out: [https://elide.io/pages/guide/v5/04-analytics.html#table-properties](https://elide.io/pages/guide/v5/04-analytics.html#table-properties)
+
+### How to i convert data of one type into another type in my semantic config?
+
+If you want to convert data from one type to another, you can provide a SQL definition for the column in the Hjson config. See below for an example of converting a Timestamp to String in 'YYYY-MM-DD' format.
+
+```
+{
+    name : createdOn
+    type : TEXT
+    definition : '''
+    FORMATDATETIME({{created_on}},'yyyy-MM-dd')
+    '''
+}
+```
+
+For more information on SQL definition please check out: [https://elide.io/pages/guide/v5/04-analytics.html#analytic-queries](https://elide.io/pages/guide/v5/04-analytics.html#analytic-queries)
+
+### how do I create fields like i would with a case statement in SQL in my semantic config?
+
+You can use CASE statement in the SQL definition for a column.    
+
+```
+{
+    name : gameType
+    type : TEXT
+    definition : '''
+    CASE WHEN {{game_type}} = 'tournament' THEN 'CHAMPIONPSHIP' ELSE {{game_type}} END
+    '''
+    friendlyName: Game Type
+}
+```
+
+For more information on case statements please check out: [https://elide.io/pages/guide/v5/04-analytics.html#column-properties](https://elide.io/pages/guide/v5/04-analytics.html#column-properties)
+
+### How do I filter on multiple fields like i would with SQL in my semantic config?
+
+To filter in a field in the semantic model, you will need to use the Elide filtering operators [https://elide.io/pages/guide/v5/11-graphql.html#filtering](https://elide.io/pages/guide/v5/11-graphql.html#filtering)
```

Comment:
I think this line can be removed as this is related to working directly with Elide API calls